ftml/src/parsing/rule/impls/block/blocks/hidden.rs

```diff
@@ -0,0 +1,60 @@
+/*
+ * parsing/rule/impls/block/blocks/hidden.rs
+ *
+ * ftml - Library to parse Wikidot text
+ * Copyright (C) 2019-2021 Wikijump Team
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+use super::prelude::*;
+
+pub const BLOCK_HIDDEN: BlockRule = BlockRule {
+    name: "block-hidden",
+    accepts_names: &["hidden"],
+    accepts_special: false,
+    accepts_newlines: false,
```

Comment:
This means that it only works inline, right? What if the user wants to hide a couple of paragraphs, for whatever misguided reason?

Reply:
Oh right this is a good catch, I forgot to set these.